src/mme-app/mmeStates/attachWfCsResp.cpp

```diff
@@ -64,4 +64,11 @@ void AttachWfCsResp::initialize()
                 actionTable.setNextState(AttachWfInitCtxtRespAttCmp::Instance());
                 eventToActionsMap.insert(pair<uint16_t, ActionTable>(CS_RESP_FROM_SGW, actionTable));
         }
+        {
+                ActionTable actionTable;
+                actionTable.addAction(&ActionHandlers::send_attach_reject);
+                actionTable.addAction(&ActionHandlers::send_s1_rel_cmd_to_ue);
+                actionTable.addAction(&ActionHandlers::abort_attach);
```

Comment:
Should we wait for s1 release complete before doing abort attach?

Reply:
S1 release in error path is implemented as fire and forget since the action we take remains the same whether we get a response or not. If there is any merit in adding wait state we can add it.